### DISCUSSION TITLE
What methods are there to render the car and background, bloom separately and keeping the shadows casted on the ground from the car  for compositing?

### DISCUSSION THREAD

**Discussion by rosstheblenderanimator** [bnpr/Malt#555] [Q&A] [answered]
![car tunnel animation wip render 8](https://github.com/bnpr/Malt/assets/146353254/b47e653b-a44a-4387-9a0b-1454e1fb0128)

**Comment by HSDSZ** [comment]
I have a shadow catcher shader for compositing. maybe helpful
![image](https://github.com/bnpr/Malt/assets/36077492/b4c9dcd9-ea19-4554-af77-d6c65b4f7e64)
[shadowcatcher.zip](https://github.com/bnpr/Malt/files/15360931/shadowcatcher.zip)

**Reply by rosstheblenderanimator** [reply]
I will give a try

**Reply by rosstheblenderanimator** [reply]
Does not work the way I wanted it to thanks anyways.

**Reply by rosstheblenderanimator** [reply, marked as answer]
Found a solution

**Reply by eballeste** [reply]
hey @rosstheblenderanimator! hi!

it would be great if your marked solution actually includes a solution so others can benefit from the help others give to you

**Reply by rosstheblenderanimator** [reply]
Here's a the solution https://github.com/bnpr/Malt/discussions/423 but a weird grey border around the foreground object.

![Uploading car tunnel animation compositing test 3.png…]()
